src/test/java/com/github/commonsrdf/dummyimpl/TripleImpl.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.commonsrdf.dummyimpl;
+
+import com.github.commonsrdf.api.BlankNodeOrIRI;
+import com.github.commonsrdf.api.IRI;
+import com.github.commonsrdf.api.RDFTerm;
+import com.github.commonsrdf.api.Triple;
+
+public class TripleImpl implements Triple {
```

Comment:
Could you add hashCode and equals implementations here and in the RDFTerm implementations (Eclipse autogenerated versions would be okay)? (See above comment about LinkedHashSet for why this would be useful even in a test set)